src/Maker/MakeController.php

```diff
@@ -55,6 +56,7 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
             $controllerClassNameDetails->getFullName(),
             'controller/Controller.tpl.php',
             [
+                'parent_class_name' => (Kernel::VERSION_ID) >= 40100 ? 'AbstractController' : 'Controller',
```

Comment:
you should detect the FrameworkBundle version actually, not the HttpKernel one

Reply:
I don't know how to do that. Which is the constant in `FrameworkBundle` that stores its version? Thanks!

Reply:
`class_exists(AbstractController::class)` should be good